pyproject.toml

```diff
@@ -36,6 +36,7 @@ marshmallow-sqlalchemy = "*"
 nanoid = "*"
 notifications-python-client = "*"
 phonenumbers = "*"
+protobuf = "^4.25"  # TODO: Pinned because version 5.27.2 failed to deploy
```

Comment:
Did you create a ticket to look into this issue? The ticket number should be in the TODO comment.

Reply:
^^^ please make that ticket and add the ticket number here